website/docs/guides/flagger-manual-gating.mdx

```diff
@@ -0,0 +1,147 @@
+	Phase CanaryPhase `json:"phase"`
+
+	// Metadata (key-value pairs) for this webhook
+	Metadata map[string]string `json:"metadata,omitempty"`
+}
+```
+
+For more information on Webhooks in Flagger, see the 
+[Flagger documentation](https://docs.flagger.app/usage/webhooks)
+
+
+## Using Flagger's load tester to manually gate a promotion
+To enable manual approval of a promotion we are going to configure the 
+`confirm-promotion` webhook to call a particular gate provided through 
+Flagger's included load tester. This is an easy way to experiment with 
+the capability using Flagger's included components. 
+
+**Important note**
+We strongly recommend that you DO NOT USE the load tester for manual gating 
+in a production environment. There is no auth on the load tester, so 
+anyone with access to the cluster would be able to open and close; and 
+the load tester has no storage, so if restarted - all gates would close. 
+
+Instead, configure these webhooks for appropriate integration with a 
+tool of your choice such Jira, Slack, Jenkins, etc.
+
+### Configure the confirm-promotion webhook
+In your Canary object, add the following in the `analysis` section:
+
+```yaml
+  analysis:
+    webhooks:
+      - name: "ask for confirmation"
+        type: confirm-promotion
+        url: http://flagger-loadtester.test/gate/check
+```
+
+This gate is closed by default.
+
+### Deploy a new version of your application
+Trigger a Canary rollout by updating your target deployment/daemonset, for 
+example by bumping the container image tag. A full list of ways to trigger 
+a rollout is available 
+[here](https://docs.flagger.app/faq#how-to-retry-a-failed-release).
+
+You can watch the progression of a Canary in Weave GitOps Enterprise (WGE) 
+through the Applications > Delivery view:
+
+![Podinfo Canary progressing](/img/pd-table-progressing.png)
+
+
+### Wait for the Canary analysis to complete
+Once the Canary analysis has successfully completed, Flagger will call the 
+`confirm-promotion` webhook and change status to `WaitingPromotion` as you 
+can see in the screenshots below:
+
+![Podinfo Canary showing Waiting Promotion - table view](/img/pd-table-waiting.png)
+
+![Podinfo Canary showing Waiting Promotion - details view](/img/pd-details-waiting.png)
+
+### Open the gate
+To open the gate and therefore confirm that you are happy for the new 
+version of your application to be promoted, we can exec into the load tester 
+container:
+
+```
+$ kubectl -n test exec -it flagger-loadtester-xxxx-xxxx sh
+
+# to open
+> curl -d '{"name": "app","namespace":"test"}' http://localhost:8080/gate/open
+```
+
+Flagger will now proceed to promote the Canary version to the primary and 
+complete the progressive delivery rollout :tada:
+
+![Podinfo Canary succeeded - full events history](/img/pd-events-gate-passed.png)
+
+![Podinfo Canary succeeded - promoting](/img/pd-table-promoting.png)
+
+![Podinfo Canary succeeded - promoting](/img/pd-table-succeeded.png)
```

Comment:
`Podinfo Canary succeeded - promoting` could be changed to `Podinfo Canary succeeded - promotion succeeded`

Reply:
copypaste laziness 😄